### DISCUSSION TITLE
Do not exit when detected failure

### DISCUSSION THREAD

**Discussion by shiwaimanong** [LizardByte/community#50] [Sunshine]
### Describe the Bug

I use Sunshine in virtual machine with nvidia a10 vpu. GPU takes a while to be ready. When set to autostart, Sunshine can not find gpu and exit.

### Expected Behavior

Sunshine keep run, even if the GPU is not detected. So Sunshine can work when the GPU is ready. It is best to provide a way to run in the background without displaying the cmd window.

### Additional Context

nvidia gamestream not support to enable shield for A10 vgpu. I hope to launch Sunshine as a remote tool when system start.

I foud this script to retry:

```
@echo off
:reboot 
cd C:\Program Files\SunshineStream
sunshine.exe
TIMEOUT /T 2
GOTO:reboot
```

### Sunshine Host Operating System and Version

Windows 10

### Architecture

64Bit

### Sunshine Version

v0.14.0

### GPU Type

nvidia

### GPU Model

A10

### GPU Driver/Mesa Version

512.78 grid

### Capture Method (Linux Only)

QEMU(KVM)

**Comment by ReenigneArcher** [comment]
Running in the background is accomplished via a service. Typically we have one, but there is a bug in v0.14.0. See #197 

The rest of this is a feature request so I will move the issue there.

**Comment by KuleRucket** [comment]
You could just use the "Restart=always/on-failure" in the systemd unit file.

EDIT: Oops it's a windows question. Sorry can help there you're on your own.
